docs/architecture/adr-045-admin-module.md

```diff
@@ -0,0 +1,142 @@
+
+# ADR 045: Admin module
+
+## Changelog
+
+- 2021/07/16: Initial draft
+
+## Status
+
+Proposed
+
+## Abstract
+
+This ADR proposes a new module - admin module, that adds simplified governance workflow with authority-based proposal acceptance. A new role is added - administrator. Admins will be capable of:
+- submitting a proposal that will automatically be accepted;
+- add/remove other admins;
+
+## Context
+
+The current proposal workflow is based on a concept of validators (token holders) who vote to accept/reject a proposal. The tallying of those votes is based on the voting power (stake-based voting) of validators and their delegators. This stake-based multi-step approach provides a high level of security necessary for mainnets.
+
+However, the specifics of testnets - often updates with new features that need to be well tested.
+
+Almost every feature and every scenario require some proposals to be accepted. For example, changing a test account's balance (thanks to blockchain technology) is not as easy as changing some number in a bank database. Any modification of a blockchain state requires a full proposal workflow with the majority of stakeholders to participate in voting. The situation is even more complicated in testnet - there's no motivation for users to participate in voting/governance. The full governance process is too long, so it slows down testing and the delivery of new features.
```

Comment:
I think in the context of testnets mainly, this makes a lot of sense.

Reply:
exactly, this is really only meant for testnet environments (altho technically a chain may want it on mainnet for things like emergency upgrades)